secio/src/codec/encode.rs

```diff
@@ -63,21 +63,15 @@ where
     type SinkError = S::SinkError;
 
     fn start_send(&mut self, item: Self::SinkItem) -> StartSend<Self::SinkItem, Self::SinkError> {
-        let capacity = item.len() + self.hmac_key.digest_algorithm().output_len;
 
-        // Apparently this is the fastest way of doing.
-        // See https://gist.github.com/kirushik/e0d93759b0cd102f814408595c20a9d0
-        let mut out_buffer = BytesMut::from(vec![0; capacity]);
+        let mut data_buf = item;
```

Comment:
Why this line, instead of renaming the function parameter or using `item` in the code below?

Reply:
Useless (just wanted to keep proto looks but does not need to be here).